flask/namespaces.py

```diff
@@ -4,6 +4,46 @@
 from app import swagger
 from flasgger.utils import swag_from
 
+from datetime import datetime
+from auth import generate_uuid
+
+@app.route("/namespaces", methods=["POST"])
+def create_namespace():
+    uuid = request.form.get("uuid")
+
+    if not uuid:
+        return jsonify({"code": 401, "message": "Unauthorized"}), 401
```

Comment:
I'm not too sure but is this the way to check if this is an authorized user? If you send a mock request with a random string in the uuid form field, doesn't this already suffice to pass this check?

Reply:
Correct. I missed the check to verify if the user belongs to the uuid. Will push a new commit.